test/integration/account_oauth_client_test.go

```diff
@@ -48,13 +50,13 @@ func TestGetOAuthClient_found(t *testing.T) {
 	}
 }
 
-func TestListOAuthClients(t *testing.T) {
+func TestOAuthClients_List(t *testing.T) {
 	createOpts := linodego.OAuthClientCreateOptions{
 		Public:      true,
 		RedirectURI: "https://example.com",
-		Label:       "test-client",
+		Label:       fmt.Sprintf("test-client-%.d", time.Now().Second()),
```

Comment:
Same as above. To prevent conflicts we could call it something like `test-client-list` 👍